volttron/platform/dbutils/mysqlfuncts.py

```diff
@@ -213,13 +219,7 @@ def setup_aggregate_historian_tables(self, meta_table_name):
     def query(self, topic_ids, id_name_map, start=None, end=None, skip=0,
               agg_type=None,
               agg_period=None, count=None, order="FIRST_TO_LAST"):
-        """This function should return the results of a query in the form:
```

Comment:
I tried to do it with an event function on sphinx but couldn't. I have add this manually. That's why added it at top level. It doesn't look bad in the resulting html since the only documentation child classes have are class level and main so its quite prominent.